mxcubeweb/core/models/adaptermodels.py

```diff
@@ -34,7 +34,10 @@ class NStateModel(HOActuatorModel):
     value: str = Field("", description="Value of nstate object")
 
 
-class HOMachineInfoModel(HOActuatorModel):
+class HOMachineInfoModel(HOModel):
```

Comment:
`pydantic 2.x` does apparently not support changing field type when overriding a model, in this case `value`